test/mock/MockBlastYield.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {YieldMode, GasMode} from "../../../contracts/interfaces/IBlast.sol";
+
+contract MockYield {
```

Comment:
the contract name should **always** be the same as the file name with the exception of test files

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/ce33daf9cf033280516e78d9ac601a8b5da5b6fb